### PR TITLE
feat: when a person fills no service fellowship summary screen should…

### PR DESCRIPTION
**Pull request by dabick14**
… not show banking slip info

### CODE REVIEW

File: web-react/src/pages/directory/display/DetailsFellowship.jsx [hunk 1]
```diff
@@ -34,24 +34,38 @@ const DetailsFellowship = () => {
...
-      return {
-        number: week,
-        filled: true,
-        banked:
-          lastFilledBanking?.length && (lastFilledBanking[i] ? true : false),
+  const check = last3Weeks()?.map((number) => {
+    if (lastFilled?.some((thing) => thing.week === number)) {
+      const inside = lastFilled?.find(({ week }) => week === number)
+      // eslint-disable-next-line no-console
+      console.log('inside', inside)
```
Comment (jaedag) [inline web-react/src/pages/directory/display/DetailsFellowship.jsx:49]: No console logs. Please delete

File: web-react/src/pages/directory/display/DetailsFellowship.jsx [hunk 2]
```diff
@@ -34,24 +34,38 @@ const DetailsFellowship = () => {
...
 
-  const check = last3Weeks()?.map((week, i) => {
-    if (lastFilledServices?.includes(week)) {
-      return {
-        number: week,
-        filled: true,
-        banked:
-          lastFilledBanking?.length && (lastFilledBanking[i] ? true : false),
+  const check = last3Weeks()?.map((number) => {
+    if (lastFilled?.some((thing) => thing.week === number)) {
```
Comment (jaedag) [inline web-react/src/pages/directory/display/DetailsFellowship.jsx:46]: Please name your variables better. What is thing? Inside? 

It's not easy to understand

**Review by jaedag — CHANGES_REQUESTED** [review]

File: web-react/src/components/DisplayChurchDetails/DisplayChurchDetails.jsx [hunk 3]
```diff
@@ -154,6 +154,8 @@ const DisplayChurchDetails = (props) => {
       shouldFill = false
     }
 
+    // eslint-disable-next-line no-console
+    console.log(props)
```
[the remark this answers is not in the document]
Comment (dabick14) [reply web-react/src/components/DisplayChurchDetails/DisplayChurchDetails.jsx:158]: Done

File: web-react/src/pages/directory/display/DetailsFellowship.jsx (hunk 2 above)
Comment (dabick14) [reply web-react/src/pages/directory/display/DetailsFellowship.jsx:46]: Changed

**Comment by dabick14** [comment]
@jaedag resolved the issues.

**Review by jaedag — APPROVED** [review]

**Pull request merged**